### PR TITLE
Bug 1125183 - Ensure we have l10n metas and recover if we don't

### PR DESCRIPTION
**Pull request by zbraniecki**
https://bugzilla.mozilla.org/show_bug.cgi?id=1125183

### CODE REVIEW

**Comment by try-server-hook** [comment]
[zbraniecki Zibi Braniecki (zbraniecki)](https://github.com/zbraniecki) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2ab159ea485b)

File: build/l10n.js [hunk 1]
```diff
@@ -72,8 +72,10 @@
       document.documentElement.dataset.noCompleteBug = true;
     }
 
-    this.ctx.registerLocales(meta.defaultLanguage,
-                             Object.keys(meta.availableLanguages));
+    var al = meta.availableLanguages ?
```
Comment (KevinGrandon) [inline build/l10n.js]: nit: I generally prefer longer "readable" variable names, especially if it's not a commonly used thing. It's more typing, but more readable imo. Would prefer ```langs``` or ```availableLangs```.

**Comment by try-server-hook** [comment]
[zbraniecki Zibi Braniecki (zbraniecki)](https://github.com/zbraniecki) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=859fb4c5c879)

File: build/l10n.js (hunk 1 above)
Comment (zbraniecki) [inline build/l10n.js]: thanks! I got lazy for a second there

**Comment by try-server-hook** [comment]
[zbraniecki Zibi Braniecki (zbraniecki)](https://github.com/zbraniecki) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=eccad8fb956c)

**Comment by try-server-hook** [comment]
[zbraniecki Zibi Braniecki (zbraniecki)](https://github.com/zbraniecki) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c45e9bdfb9d0)

**Pull request merged**